### DISCUSSION TITLE
Python 3 only

### DISCUSSION THREAD

**Discussion by ArkNieckarz** [emmett-framework/emmett#306] [Ideas]
Would be great to drop support for Python 2 altogether. CherryPy and Django have already done that.
And update required packages to work with Python 3.9+.

Currently I can't run this on Python 3.8+ because of: in python 3.8 cgi.escape will be removed, need to see: Use html.escape instead of cgi.escape on python 3.8+

Also rename project name on github from emmett-framework /emmett to weppy, or change weppy to emmett-framework.
The two are confusing and misleading.

**Comment by gi0baro** [comment]
@ArkNieckarz actually Emmett is already Python 3 only.

While weppy is still Python 2 compatible, its successor supports only Python 3.7 and above versions. The change into package name was exactly driven to avoid confusion: keep weppy and existing projects based on it fully compatible and working, while proceeding with new features and changes on Emmett.

You can check the [changelog](https://github.com/emmett-framework/emmett/blob/master/CHANGES.md#version-20) for the 2.0 and also the [upgrade guide](https://emmett.sh/docs/2.1.x/upgrading#version-20) for further details.

Mind that while the weppy website and project are kept intact (you can find the [1.x branch on this repo](https://github.com/emmett-framework/emmett/tree/1.x)) they won't see any upgrades in the future, only security fixes are guaranteed.
